### PR TITLE
fix #31528 r3sub.com

### LINKED ISSUE #31528: r3sub.com
### Issue URL (Anti Adblock Script)
[https://r3sub.com/show.php?id=A4iF4813298](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fr3sub.com%2Fshow.php%3Fid%3DA4iF4813298)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.0.11
Browser: | Chrome
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Self-destructing third-party cookies (2880), Hide your Referrer from third-parties
Filters: | EasyList China,<br>EasyPrivacy,<br>AdGuard Base,<br>Adblock Warning Removal List,<br>(Obsolete) Anti-Adblock Killer | Reek,<br>Fanboy's Enhanced Tracking List,<br>ChinaList+EasyList,<br>AdGuard Tracking Protection

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/31528

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -851,6 +851,8 @@ javfree.club#%#(function() { var w_open = window.open, regex = /agxclick\.com/;
 ! https://github.com/AdguardTeam/AdguardFilters/issues/33453
 85tube.com##.block-video > .table
 ||85tube.com/player/html.php?aid=*_html&video_id=*&*&referer
+! https://github.com/AdguardTeam/AdguardFilters/issues/31528
+r3sub.com##.container a[target="_blank"] > img
```
Comment (BlazDT) [inline EnglishFilter/sections/foreign.txt]: Can you specify the rule a bit more please. Sadly the site is not opening for me.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: Try Hong Kong VPN to open this site. Now this rule is redundant cause i can't reproduce ad block.

Comment (BlazDT) [reply EnglishFilter/sections/foreign.txt]: That means we can close this pull request?

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: Yes, now it's clean for me.

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: Yesterday i can't reproduce it, but today i see just a one banner.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**